testing/polipo/APKBUILD

```diff
@@ -0,0 +1,31 @@
+# Contributor: Alexander Georgievskiy <galeksandrp@gmail.com>
+# Maintainer:
+pkgname=polipo
+pkgver=1.1.1
+pkgrel=0
+pkgdesc="The Polipo caching HTTP proxy"
+url="https://github.com/jech/$pkgname"
+arch="all"
+license="MIT"
+depends=""
+makedepends="texinfo"
+install=""
+subpackages=""
+source="$url/archive/$pkgname-$pkgver.tar.gz"
```

Comment:
personal nitpick here: i would prefer https://github.com/jech/$pkgname/archive/... just for ease of understanding $sources in future